server/storage/storage_impl.go

```diff
@@ -374,12 +336,15 @@ func (s *metaStorageImpl) GetTable(ctx context.Context, clusterID uint32, schema
 func (s *metaStorageImpl) ListTables(ctx context.Context, clusterID uint32, schemaID uint32) ([]*clusterpb.Table, error) {
 	tables := make([]*clusterpb.Table, 0)
 	nextID := uint64(0)
-	endKey := makeTableKey(clusterID, schemaID, math.MaxUint64)
+	endKey := makeTableKey(s.rootPath, clusterID, schemaID, math.MaxUint64)
```

Comment:
Maybe we should define a method in the etcdutil package to avoid the duplicate codes:
```go
ForKeysInRange(client etcdv3.client, startKey, endKey string, batch_size uint64, do func(key, value) -> error) -> error
```